5.1.thats_the_way.py

```diff
@@ -0,0 +1,13 @@
+import os
+from os import path
+
+def thats_the_way(dir_path, start_with):
+    L = []
```

Comment:
Rename 'L' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing